src/scenes/Alerts.jsx

```diff
@@ -0,0 +1,241 @@
+        this.handleSlide = this
+            .handleSlide
+            .bind(this);
+        this.handleSubmit = this
+            .handleSubmit
+            .bind(this);
+        this.handleDelete = this
+            .handleDelete
+            .bind(this);
+    }
+    handleSlide(value) {
+        const {form} = this.state;
+        form.radius = value;
+        this.setState({form, error:false});
+    }
+
+    handleAddress(e) {
+        const {form} = this.state;
+        form.address = e.target.value;
+
+        this.setState({form, error:false});
+    }
+
+    handleSubmit(e){
+        const {address, radius} = this.state.form;
+        e.preventDefault();
+        
+        this.setState({loading:true});
+        
+        api.post('/alert',{
+            address, 
+            radius: this.state.slider.max + this.state.slider.min- radius
+        })
+        .then(() => this.getAlerts())
+        .finally(()=>{
+            this.setState({
+                loading: false,
+                 form:{
+                    radius: 3,
+                    address: ''
+            }});
+        })
+        .catch(error => {
+            this.setState({error})
+            console.error(error);
+        })
+    }
+
+    handleDelete(alertId) {
+        api.delete('/alert/'+alertId)
+        .then(() => this.getAlerts())
+    }
+
+    componentDidMount() {
+        this.getAlerts();
+        document.getElementById('homeAddress').focus();
+    }
+
+    getAlerts() {
+        return api
+        .get('/alert')
+        .then(result => {
+            let transparentLayer = leaflet.geoJSON();
+            let fixedBounds = this.state.bounds;
+            if(result.data.length > 0){
+                result.data.map((alert)=>{
+                    leaflet.geoJSON(alert.geom).addTo(transparentLayer);    
+                  })
+                  const layerBounds = transparentLayer.getBounds();
+                  fixedBounds= [layerBounds._southWest, layerBounds._northEast];
+                }
+         this.setState({bounds: fixedBounds, alerts:result.data});
+         
+        })
+        .catch(error => this.setState({error}))
+    }
+    
+    render() {
+        const {alerts, form, error, loading, bounds} = this.state;
+        const {me} = this.props;
+
+        const sliderText = {};
+        _.map(new Array(this.state.slider.max), (obj, i)=>{
+            sliderText[i+1]= this.state.slider.max- i  + " ק״מ";
+        })
+        // unauthenticatd
+        if (error && error.response && error.response.status === 403) {
+            return <Redirect to="/sign/in" />
+        }
+        return <React.Fragment>
+            <Navigation me={me}/>
+            <div className="container widedialog">
+                <form className="rectangle" onSubmit={this.handleSubmit}>
+                    <h5 className="container-title">התראה חדשה</h5>
+                    {error && <div className="alert alert-danger" role="alert">הכתובת לא נמצאה</div>}
```

Comment:
alert has few states. they must be different states.
noAddressError, noAuthentifactedError can help